site/en/docs/devtools/javascript/breakpoints/index.md

```diff
@@ -104,9 +158,9 @@ To set a DOM change breakpoint:
 4.  Hover over **Break on** then select **Subtree modifications**, **Attribute modifications**, or
     **Node removal**.
 
-{% Img src="image/admin/PlPfY25q2BTOrc69IVDj.png", alt="The context menu for creating a DOM change breakpoint.", width="800", height="577" %}
+{% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/XsXVifCHA03yWaSJHeUR.png", alt="The context menu for creating a DOM change breakpoint.", width="800", height="756" %}
 
-**Figure 5**: The context menu for creating a DOM change breakpoint
+This example shows the context menu for creating a DOM change breakpoint.
```

Comment:
Not related to breakpoint pane changes, but this session missing the guidance on where to manage this breakpoint. - Elements panel > DOM breakpoints pane, Sources panel > DOM Breakpoints side pane.